inc/settings.php

```diff
@@ -137,6 +137,7 @@ public function get_default_option_values() {
 			'woocommerce_gzd_display_checkout_back_to_cart_button'          => 'no',
 			'woocommerce_gzd_display_checkout_table_color'                  => '#eeeeee',
 			'woocommerce_enable_guest_checkout'                             => 'yes',
+			'free_gifts_for_woocommerce_position'                           => 'after_cart_items',
```

Comment:
@vlad-dedikov is this really necessary? This option is not used on any part of the Lite or PRO plugins.